src/Pages/Detail/Detail.scss

```
@@ -0,0 +1,292 @@
+@import '../../Styles/common.scss';
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,800;1,800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Nanum+Gothic:wght@800&display=swap');
+
+.content{
+  padding: 40px 0 0 0;
+  width: 1200px;
+  margin: 0 auto;
+  .item_photo_info_sec{
+    display: flex;
+    justify-content: center;
+    padding: 30px 30px;
+    .item_photo_view_box{
+      margin-top: 200px;
```

```suggestion
.content {
  padding-top: 40px;
  max-width: 1200px;
  margin: 0 auto;
  
  .item_photo_info_sec{
    display: flex;
    justify-content: center;
    padding: 30px;
    
    .item_photo_view_box{
      margin-top: 200px;
```
- 셀렉터 뒤 중괄호는 한칸 띄워주세요.
- padding이 상단에만 필요하다면 padding-top을 사용해주세요.
- 새로운 셀렉터는 한줄 띄워주세요.
- 역시 네 방향 모두 같은 padding 값이 필요하다면 단축속성 써주세요.
- 가로 길이를 저렇게 길게 주지마시고, 최대 값만 주되 자식 요소들의 너비의 합으로 부모의 길이가 구성될 수 있도록 해주세요.